### DISCUSSION TITLE
Wrong instructions on v3 documentation

### DISCUSSION THREAD

**Discussion by psjavi** [livewire/livewire#7170] [7. Bugs (archived)]
### Livewire version

v3

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

Hi, in the docs you said to add @livewireScripts but it's wrond due to the migration docs that you said to move from   @livewireScripts to  @livewireScriptConfig 

![image](https://github.com/livewire/livewire/assets/4563033/f70cb9c8-ca91-4095-97d4-8fdc30cec60c)

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@psjavi The instructions are correct. 

You should only use livewireScriptConfig if you are bundling Alpine and Livewire in your app.js file. 

Otherwise you should add livewireScripts to your layout so Alpine is available on pages where there is no Livewire (if you need it).

Hope this helps!

**Comment by psjavi** [comment]
Thanks for the info @joshhanley

**Reply by joshhanley** [reply]
No worries! 🙂
